README.md

```diff
@@ -1 +1,2 @@
-# jpn-lang-api
\ No newline at end of file
+# jpn-lang-api
```

Comment:
Can you update correct project name?

Reply:
i have already updated the project name.